build/linux/dist/install.sh

```diff
@@ -6,7 +6,7 @@
 # If called with the "-u" option, it will undo the changes.
 
 # Resource name to use (including vendor prefix)
-RESOURCE_NAME=arduino-arduinoide
+RESOURCE_NAME=cc.arduino.arduinoide
```

Comment:
Yes, it gets added later in the script (so the ID can be used by multiple things)